### ISSUE TITLE
Cannot obtain information required for properly filing a bug report: Qt version

### ISSUE THREAD

**Issue by mk-pmb** [PrismLauncher/PrismLauncher#1688]
### Operating System

Linux

### Version of Prism Launcher

Prism Launcher 7.2

### Version of Qt

I wish I could check.

### Description of bug

The download window is blocking access to the main window, so I can't look up the Qt version to properly report a bug about my download experience. Not willing to sacrifice all my download progress for that.

### Steps to reproduce

1. Download a modpack from a really slow server or via very slow network.
2. Follow the bug template instructions to try and find Qt version.
3. Discover you can't (main window won't react).

### Suspected cause

Modal windows. We have just way too many. Why should downloading files for one instance prevent me from managing the mods or saves of another instance?

I tried launching a second Prism process but all it did was activate and raised the download window.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
Thank you for reporting this but this looks like a duplicate of https://github.com/PrismLauncher/PrismLauncher/issues/1096
Is not as exact as this one but solves the same problem.
Also, I do not see a reason why opening an issue on GitHub can't wait until your download finishes.

**Comment by mk-pmb** [comment]
> why opening an issue on GitHub can't wait until your download finishes.

Busy schedule and drain on motivation. In the first attempt, the change between the two phases of being stuck seems to have occurred sometime between 8 hours and 14 hours. I was already some hours into my attempt to reproduce when I discovered that I couldn't find my note with the Qt version anymore. At that point I figured that it would be beneficial to lower the burden for reporting this kind of bug.
